source/backend/api/Services/PropertyService.cs

```diff
@@ -191,6 +192,36 @@ public PropertyManagementModel UpdatePropertyManagement(PimsProperty property)
             return GetPropertyManagement(newProperty.Internal_Id);
         }
 
+        public IList<PimsPropPropActivity> GetManagementActivities(long propertyId)
+        {
+            _logger.LogInformation("Getting property management activities for property with id {propertyId}", propertyId);
+            _user.ThrowIfNotAuthorized(Permissions.ManagementView, Permissions.PropertyView);
+
+            return _propertyRepository.GetManagementActivitiesByProperty(propertyId);
+        }
+
+        public bool DeleteManagementActivity(long propertyId, long managementActivityId)
+        {
+            _logger.LogInformation("Deleting Management Activity with id {managementActivityId} from property with Id {propertyId}", managementActivityId, propertyId);
+            _user.ThrowIfNotAuthorized(Permissions.ManagementDelete);
+
+            var propertyManagementActivity = _propertyRepository.GetManagementActivityById(managementActivityId);
+            if(propertyManagementActivity.PropertyId != propertyId)
+            {
+                throw new BadRequestException($"PropertyManagementActivity with Id: {managementActivityId} and PropertyId {propertyId} doesn't exists");
```

Comment:
nit: should be "doesn't exist" (with no s)

Reply:
updated